### PR TITLE
media-libs/openexr: fix x86 musl build

### PR DESCRIPTION
**Pull request by cfuga**
media-libs/openexr: fix x86 musl build

This upstream commit breaks x86 32-bit with musl, because it assumes glibc == linux.

https://github.com/AcademySoftwareFoundation/openexr/commit/7e4630f9773810297831b9a31c6fecab51623e12

These patches extend the upstream fix to all non-glibc Linux systems.

Signed-off-by: Cristian Othón Martínez Vera <cfuga@cfuga.mx>

### CODE REVIEW

**Comment by waebbl** [comment]
Have you tried to upstream these patches?

**Comment by cfuga** [comment]
> Have you tried to upstream these patches?

The patch just landed in upstream.

#[https://github.com/AcademySoftwareFoundation/openexr/commit/fe463369d1c183e296024f58c4c0f2a4bd8aabc2](url)

File: media-libs/openexr/files/openexr-3.1.5-musl-i386.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--- a/src/lib/Iex/IexMathFpu.cpp
```
Comment (thesamesam) [inline media-libs/openexr/files/openexr-3.1.5-musl-i386.patch:1]: Please send this upstream (and include the context from the PR message at the top of this patch).

**Comment by cfuga** [comment]
Done! My patch was just accepted by upstream. Sorry, I sent it before @thesamesam 's review and it doesn't include a link to this PR.

[https://github.com/AcademySoftwareFoundation/openexr/pull/1470](Fix build with i386 and musl libc)

**Comment by Jamim** [comment]
_Hello @thesamesam,_

Since the issue was fixed in the upstream, but the affected versions were left in the Portage tree, I believe this PR is still worth merging.
Alternatively, you may drop `3.1.5-r1` and `3.1.7`.

_Thanks!_

**Comment by thesamesam** [comment]
We can't clean up yet as stabilisation isn't done. Anyway, thanks for the ping!

**Pull request closed without merging**